Flow.Launcher/MainWindow.xaml.cs

```diff
@@ -555,5 +557,11 @@ public void InitializeColorScheme()
                 ModernWpf.ThemeManager.Current.ApplicationTheme = ModernWpf.ApplicationTheme.Dark;
             }
         }
+
+        private void QueryTextBox_KeyUp(object sender, KeyEventArgs e)
+        {
+            if(_viewModel.QueryText != QueryTextBox.Text)
+                _viewModel.QueryText = QueryTextBox.Text;
```

Comment:
lol do you want to change the binding mode to oneway?

Reply:
It looks like a oneway but it works twoway.